File: frontend/src/features/actividades/generar_estudio_titulos/models/generar_estudio_titulos.ts
export interface GenerarEstudioTitulos {
  id_generar_estudio_titulos: number;
  id_expediente: number;
  observaciones?: string | null;
  enviar_reparo: boolean;
  is_active: boolean;
  row_status: boolean;
  created_by: number;
  created_date: string;
  modified_by?: number | null;
  modified_date?: string | null;
}

File: backend/Multibanca.Domain.Models/Multibanca/generar_estudio_titulos.cs
using Common.Domain.Models;

namespace Multibanca.Domain.Models.Multibanca
{
    public class generar_estudio_titulos:base_auditoria
    {
        public int id_generar_estudio_titulos { get; set; }
        public long id_expediente { get; set; }
        public string? observaciones { get; set; }
        public bool enviar_reparo { get; set; }
    }
}
